onix/couple/couple_openmc.py

```diff
@@ -913,7 +916,10 @@ def _pass_nuclide_densities(self, cell_dict, bucell_dict):
                 onix_nucl = utils.openmc_name_to_onix_name(nucl)
                 # if nucl is one of the initial non-zero initial nuclide, pass the density
                 if nucl in init_nucl:
-                    onix_dens_dict[onix_nucl] = openmc_dens_dict[nucl][1]
+                    if ((int(openmc.__version__.split('.')[1]) == 13) and (int(openmc.__version__.split('.')[2]) >= 1)) or (int(openmc.__version__.split('.')[1]) > 13):
+                        onix_dens_dict[onix_nucl] = openmc_dens_dict[nucl]
+                    else:
+                        onix_dens_dict[onix_nucl] = openmc_dens_dict[nucl][1]
```

Comment:
```suggestion
                    if isinstance(onix_dens_dict, dict):
                        onix_dens_dict[onix_nucl] = openmc_dens_dict[nucl]
                    else:
                        onix_dens_dict[onix_nucl] = openmc_dens_dict[nucl][1]
```

I think this makes the purpose clearer and is more readable. I believe it achieves the same functionality and may be more robust towards future changes of opened.

Reply:
While I agree with the sentiment of the explicit version check being quite unwieldy I think we should at least leave a comment referencing where this is coming from. 
Regarding the check itself, I think it needs to be slightly different to actually work as intended. And if our goal is to eliminate the unwieldy expression we might as well save the result in a variable and reuse it for the criticality, e.g. like this: 
```suggestion
                    # OpenMC changed the format of material.get_nuclide_atom_densities() from version 0.13.1 onwards
                    if isinstance(openmc_dens_dict[nucl], tuple):
                        onix_dens_dict[onix_nucl] = openmc_dens_dict[nucl][1]
                        old_openmc = True
                    else:
                        onix_dens_dict[onix_nucl] = openmc_dens_dict[nucl]
                        old_openmc = False
```